tools/primreq.ml

```diff
@@ -40,16 +40,15 @@ let scan_info cu =
 
 let scan_obj filename =
   let ic = open_in_bin filename in
-  let buffer = String.create (String.length cmo_magic_number) in
+  let buffer = Bytes.create (String.length cmo_magic_number) in
```

Comment:
@drbo There are some good tutorials online about `git rebase --interactive`. The high level explanation is that since your branch and the branch you want to merge into have both evolved independently, there'll be conflicts and the merge is guaranteed to be messy. By doing a rebase, git reapplies each of the commits in your branch on top of the current master branch, so you get to deal with conflicts one at a time and even do some cleanup like merging some of your commits together. Then once that's done you can open a Pull Request and it's easier to review and the merge is clean and painless. I can help you with this if you have Skype or Hangouts.